catalog/dags/providers/provider_api_scripts/europeana.py

```diff
@@ -163,7 +163,7 @@ def _get_creator(self, data: dict) -> str | None:
         if isinstance(creators, list):
             if not creators:  # Explicitly handle empty list
```

Comment:
It would be better to move the `if not creators` condition higher, so we return `None` both for an empty list and an empty string (`""`).